packages/valory/skills/abstract_round_abci/base.py

```diff
@@ -181,10 +181,15 @@ class BaseTxPayload(metaclass=_MetaPayload):
     @property
     def data(self) -> Dict[str, Any]:
         """Data"""
-        excluded = ["sender", "round_count", "id_", "_metaclass_registry_key"]
+        excluded = ["sender", "round_count", "id_"]
         return {k: v for k, v in asdict(self).items() if k not in excluded}
 
-    # TODO: refactor - these methods are not strictly needed
+    @property
+    def values(self) -> Tuple[Any, ...]:
```

Comment:
This is more useful than `data` as it is an immutable object. The `data` property can be removed in a next round as we only use it internally in this file.